test/package/browser/template/test/package.test.ts

```diff
@@ -0,0 +1,27 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('NPM package', () => {
+  test('can be imported and provides access to Ably functionality', async ({ page }) => {
+    const pageResultPromise = new Promise<void>((resolve, reject) => {
+      page.exposeFunction('onResult', (error: Error | null) => {
+        if (error) {
+          reject(error);
+        } else {
+          resolve();
+        }
+      });
+    });
+
+    await page.goto('/');
+
+    let caughtError: Error | null;
+    try {
+      await pageResultPromise;
+      caughtError = null;
+    } catch (error) {
+      caughtError = error;
+    }
+
+    expect(caughtError).toBe(null);
```

Comment:
why would we do this instead of simple `await`ing the promise and letting it fail the test if it throws an error?

Reply:
I seem to remember that something was grumbling when I tried this before, but whatever it was I can't reproduce it now. Have changed it.